doc/readme/EXAMPLES.md

```diff
@@ -94,6 +94,14 @@ You can render it with
 which is also part of `make doc`.
 The target might not be available if the requirements in `doc/cookbook/requirements.txt` are not satisfied (in particular Sphinx), or if the meta examples are disabled.
 
+Even if all the requirements are satisfied and the following error is visible
+
+    make: *** No rule to make target 'cookbook'. Stop.
```

Comment:
mmm i would rather talk about that you should check the `SPHINX_EXECUTABLE` in the CMakeCache.txt and make sure that it's an actual path. if not then... and then write about the -DSPHINX_EXECUTABLE cmake option

Reply:
But if it is a real path then the error would never come right?

Reply:
Added.

Reply:
this is way too specific for make target. there are many other generators (ninja, xcode, visual studio etc). Just write that in case the cookbook target is still missing inspect the followings....